source/assets/stylesheets/styles/_page_content.scss

```diff
@@ -1,20 +1,22 @@
 %page-content-margins {
-  margin-top: 85px;
+  margin-top: 87px;
   padding-bottom: 30px;
 
   @include breakpoint($tablet-breakpoint) {
-    margin-top: 54px;
+    margin-top: 93px;
     padding-bottom: 50px;
   }
 
   @include breakpoint($desktop-breakpoint) {
-    margin-top: 64px;
+    margin-top: 100px;
   }
+
 }
 
 .page-content {
   @extend %page-content-margins;
   background-color: $light-gray;
+
```

Comment:
There's a couple of extra blank lines we should probably remove here for consistency